src/state/AccountCreationState.ts

```diff
@@ -59,11 +69,13 @@ export class AccountCreationState implements IState{
     public async onStart(): Promise<void> {
         const currentArgv = this.cliService.getCurrentArgv();
         const async = currentArgv.async;
-        this.logger.info(`Starting Account Creation state in ${async ? `asynchronous` : `synchronous`} mode...`, this.stateName);
-
+        this.blacklistedAccountsCount = await this.getBlacklistedAccountsCount();
+        this.logger.info(`Starting Account Creation state in ${async ? `asynchronous` : `synchronous`} mode... with ${this.blacklistedAccountsCount} blacklisted accounts`, this.stateName);
```

Comment:
Make this not should 0 blacklisted accounts if the feature is not even turned on. It will add unneeded confusion otherwise.